cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -516,7 +523,7 @@ func generateInitramfsMounts() error {
 }
 
 func unlockIfEncrypted(name string) (string, error) {
-	device := filepath.Join("/dev/disk/by-label", name)
+	device := filepath.Join(diskByLabel, name)
 	encdev := device + "-enc"
```

Comment:
I wonder if  we should be more explicit and do the boring thing:

encdev := filepath.Join(devDiskByLabelDir, name+"-enc")